README.md

```diff
@@ -258,7 +258,7 @@ Required **function**. Similar to other two components' `children`. Receive back
 ##### - willLeave?: (styleThatLeft: TransitionStyle) => ?Style
 Optional. Defaults to `() => null`. **The magic sauce property**.
 
-- `styleThatLeft`: the e.g. `{key: ..., data: ..., value: ...}` object from the `styles` array, identified by `key`, that was present during a previous render, and that is now absent, thus triggering the call to `willLeave`.
+- `styleThatLeft`: the e.g. `{key: ..., data: ..., style: ...}` object from the `styles` array, identified by `key`, that was present during a previous render, and that is now absent, thus triggering the call to `willLeave`. Note that the style property is exactly what you passed in `styles`, and is not interpolated. For example, if you passed a spring for `x` you will receive an object like `{x: {stiffness, damping, val, precision}}`.
```

Comment:
Humm I intended this object to be opaque. The main use-case is you using it to return a value like so: `{key: ..., style: {x: styleThatLeft.x, y: someChangedValue}}`. Do you need to introspect it?